facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -52,7 +49,11 @@ object LiveFapiFrontPress extends FapiFrontPress {
   def pressByPathId(path: String): Future[Unit] =
     getPressedFrontForPath(path)
       .map { pressedFront => S3FrontsApi.putLiveFapiPressedJson(path, Json.stringify(Json.toJson(pressedFront)))}
-      .asFuture.map(_.fold(e => throw new RuntimeException(s"${e.cause} ${e.message}"), _ => ()))
+      .asFuture.map(_.fold(
+        e => {
```

Comment:
consider changing this to an onComplete with a case Failure/case Success just to make it more readable